docs/en/operation/Metrics-System.md

```diff
@@ -119,11 +120,11 @@ such as data transferred or the number of RPC invocations.
 
 Metrics in Alluxio have the following format for master node metrics:
 
-master.[metricName].[tag1].[tag2]...
+Master.[metricName].[tag1].[tag2]...
 
 Metrics in Alluxio have the following format for non-master node metrics:
 
-[processType].[hostName].[metricName].[tag1].[tag2]...
+[processType].[metricName].[tag1].[tag2]...[hostName]
```

Comment:
can we put this in a code block? 

````
```
```
````

Reply:
Done, thanks